guide/payments/introduction.md

```diff
@@ -9,6 +9,14 @@ main_classes: -no-top-padding
 image: https://bitcoin.design/assets/images/guide/payments/payments-preview.jpg
 ---
 
+<!--
+
+Illustration source
+
+-
```

Comment:
Nit -- did you intend to include an illustration source file here? All I see is the surrounding comment tag.

Reply:
I just got in a habit of adding that there, not sure where the file is for that header @GBKS did you do this one?

Reply:
@johnsBeharry did that one.